framework/doc/content/source/convergence/FEProblemConvergence.md

```diff
@@ -0,0 +1,74 @@
+# FEProblemConvergence
```

Comment:
"NonlinearConvergence" might be a bit misleading because a lot of other convergences apply to the nonlinear system. By the way, I'm really planning for users to not really use this Convergence - it's more of just the default. Maybe I could/should enforce that, but I'm not sure how it'd be done (can it make one if not registered?). Eventually, I hope to decompose this kind of convergence into pieces.

Are we able to specify different convergence criteria for multi-systems right now? Example? If it's not a current capability, I suggest a later step.

Reply:
Actually ping-pong is fine. It's a fair concept to define based on the residual norm evolution

Reply:
I think NonlinearDefaultConvergence would be fine - having "Residual" in the name has my original concern: it does more than look at the residual.